packages/node/src/constant.ts

```diff
@@ -22,3 +22,5 @@
 import { SOLID_CLIENT_AUTHN_KEY_PREFIX } from "@inrupt/solid-client-authn-core";
 
 export const KEY_REGISTERED_SESSIONS = `${SOLID_CLIENT_AUTHN_KEY_PREFIX}registeredSessions`;
+// Currently required, pending a response to https://github.com/panva/node-openid-client/discussions/430.
+export const TMP_ARBITRARY_SECRET = "some arbitrary secret";
```

Comment:
Perhaps make this value: 
```suggestion
export const TMP_ARBITRARY_SECRET = "arbitrary-secret-see-issue-XXX";
```

Or something such that if you see that client secret crop up, you know what it means. (it's self-descriptive)

Reply:
Fortunately I was an idiot and didn't read the documentation well enough, and this bogus client secret is no longer necessary.